frontend/src/pages/Clusters/Clusters.js

```diff
@@ -36,14 +37,35 @@ import Actions from './Actions';
 import Details from "./Details";
 import { wizardShow } from '../Configure/Configure';
 
+function updateClusterList() {
+  const selectedClusterName = getState(['app', 'clusters', 'selected']);
+  const oldStatus = getState(['app', 'clusters', 'selectedStatus']);
+
+  ListClusters((clusterList) => {
+    if(selectedClusterName)
+    {
+      const selectedCluster = findFirst(clusterList, c => c.clusterName === selectedClusterName);
+      if(selectedCluster)
+      {
+        if(oldStatus !== selectedCluster.clusterStatus)
+          setState(['app', 'clusters', 'selectedStatus'], selectedCluster.clusterStatus);
+
+        if(oldStatus === 'CREATE_IN_PROGRESS' && selectedCluster.clusterStatus === 'CREATE_COMPLETE')
```

Comment:
Shouldn't this also do updates? i.e. in the case where you update to add `SSMManagedInstanceCore` policy you'd want filesystem and shell buttons to populate.